src/common/components/modals/DeleteModal.vue

```diff
@@ -11,9 +11,10 @@
                     @confirm="handleConfirm"
     >
         <template #body>
-            <p class="delete-modal-content">
+            <p v-if="contents" class="delete-modal-content">
```

Comment:
```suggestion
            <p v-if="contents || $scopedSlots.default" class="delete-modal-content">
```

default slot이 있는 경우에도 보이도록 해주어야 사이드이펙트가 나지 않습니다~